js/tweek-client/src/TweekManagementClient/TweekManagementClient.ts

```diff
@@ -274,8 +275,65 @@ export default class TweekManagementClient implements ITweekManagementClient {
     return this._fetch(requestUrl, config).then(noop);
   }
 
+  getExternalApps(): Promise<ExternalApp[]> {
+    const url = `${this.config.baseServiceUrl}/api/v2/apps`;
+    return this._fetch(url).then(toJson);
+  }
+
+  getExternalApp({ appId }: { appId: string }): Promise<ExternalApp> {
+    const url = `${this.config.baseServiceUrl}/api/v2/apps/${appId}`;
+    return this._fetch(url).then(toJson);
+  }
+
+  createExternalApp(appData: { name: string; permissions: Array<string> }): Promise<{ secret: string; appId: string }> {
```

Comment:
can you please export types instead of inlining them?
```suggestion
export type ExternalApp = { name: string; permissions: Array<string> }
export type ExternalAppResponse = { secret: string; appId: string }

  createExternalApp(appData: ExternalApp): Promise<ExternalAppResponse> {
```